hazelcast/src/main/java/com/hazelcast/client/impl/MemberImpl.java

```diff
@@ -70,94 +68,93 @@ public String getStringAttribute(String key) {
 
     @Override
     public void setStringAttribute(String key, String value) {
-        notSupportedOnClient();
+        throw notSupportedOnClient();
     }
 
     @SuppressFBWarnings(value = "NP_BOOLEAN_RETURN_NULL", justification = "null means 'missing'")
     @Override
     public Boolean getBooleanAttribute(String key) {
-        final Object attribute = getAttribute(key);
+        Object attribute = getAttribute(key);
         return attribute != null ? Boolean.valueOf(attribute.toString()) : null;
     }
 
     @Override
     public void setBooleanAttribute(String key, boolean value) {
-        notSupportedOnClient();
+        throw notSupportedOnClient();
     }
 
     @Override
     public Byte getByteAttribute(String key) {
-        final Object attribute = getAttribute(key);
+        Object attribute = getAttribute(key);
         return attribute != null ? Byte.valueOf(attribute.toString()) : null;
     }
 
     @Override
     public void setByteAttribute(String key, byte value) {
-        notSupportedOnClient();
+        throw notSupportedOnClient();
     }
 
     @Override
     public Short getShortAttribute(String key) {
-        final Object attribute = getAttribute(key);
+        Object attribute = getAttribute(key);
         return attribute != null ? Short.valueOf(attribute.toString()) : null;
     }
 
     @Override
     public void setShortAttribute(String key, short value) {
-        notSupportedOnClient();
+        throw notSupportedOnClient();
     }
 
     @Override
     public Integer getIntAttribute(String key) {
-        final Object attribute = getAttribute(key);
+        Object attribute = getAttribute(key);
         return attribute != null ? Integer.valueOf(attribute.toString()) : null;
     }
 
     @Override
     public void setIntAttribute(String key, int value) {
-        notSupportedOnClient();
+        throw notSupportedOnClient();
     }
 
     @Override
     public Long getLongAttribute(String key) {
-        final Object attribute = getAttribute(key);
-        return attribute != null ? Long.getLong(attribute.toString()) : null;
```

Comment:
`Long.getLong()` tries to read from System properties, not from the given parameter.